compiler/src/dotty/tools/dotc/typer/Typer.scala

```diff
@@ -2279,6 +2288,34 @@ class Typer extends Namer
     (stats1, finalCtx)
   }
 
+  /** Tries to adapt NotNullInfos from creation context to the DefTree,
+   *  returns whether the adaption is successed. The adaption only success if the
```

Comment:
... returns whether the adaption took place. An adaption only takes place if the

Reply:
Done